package.json

```diff
@@ -2,21 +2,32 @@
   "name": "ryderserial-proto",
   "version": "0.0.1",
   "description": "",
-  "main": "src/index.js",
+  "main": "out/index.js",
```

Comment:
I think we should make it build to a `dist` folder including the typedefs and untrack the folder. That would be great for downstream implementations.

Reply:
deal.

Reply:
Question — how does this work for npm installation? 

If I `npm i ryderserial-proto` (though I know currently we point to github repo/branch itself) and `dist/index.js` is untracked despite being the main entry point... how is my client able to access this folder?

I'm new to distributing npm libraries so sorry if this is a stupid question.